packages/backend/src/database/migrations/20240314104022_migrate_space_share_table_data.ts

```diff
@@ -0,0 +1,178 @@
+import {
+    convertOrganizationRoleToProjectRole,
+    convertProjectRoleToSpaceRole,
+    getHighestProjectRole,
+} from '@lightdash/common';
```

Comment:
Migrations can not import code as it might change over time. We need to copy those utils to this file.